File: eclipse/idm.project.tests/ressources/expected/python/colSum.py
import pandas as pd
df = pd.read_csv( "/home/simonlecordier/workspace/m1/IDM/IDM-Projet/eclipse/idm.project.tests/ressources/input/fruits.csv" )
df.at[6, "Article"] = "Benefice Total"
df.at[6, "Stock"] = df.sum(axis=0)["Benefice"]
df.to_csv("/home/simonlecordier/workspace/m1/IDM/IDM-Projet/eclipse/idm.project.tests/ressources/output/colSumPython.csv", index=False)

File: eclipse/idm.project.tests/ressources/expected/r/colSum.R
# Creates a new DataFrame from a CSV File
df = read.csv(file = "C:/Users/Leloup/Documents/FAC/M2/IDM/Git/IDM-Projet/eclipse/idm.project.tests/ressources/input/fruits.csv")
#inserts a value at a cell
df[6+1,"Article"] = "Benefice Total"

#inserts a value at a cell
df[6+1,"Stock"] = sum(as.numeric(df$"Benefice"),na.rm = TRUE)

write.csv(df,"C:/Users/Leloup/Documents/FAC/M2/IDM/Git/IDM-Projet/eclipse/idm.project.tests/ressources/output/colSumR.csv",quote = FALSE,row.names = FALSE)

